dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05380ChangeContainerPathToAbsoluteTest.java

```diff
@@ -514,4 +535,35 @@ public void whenSiteChangeNameBeforeRunUT() throws IOException, DotDataException
 
         assertEquals(count, 1);
     }
+
+    /**
+     * Method to Test: {@link Task05380ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with relative path container in Sidebar
+     * Should: Should turn it into a Absolute Path, using the template's host
+     */
+    @Test
+    public void whenTemplateLayoutHasRelativePathAndSideBar() throws IOException, DotDataException, DotSecurityException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6309437)